soci/artifacts.go

```diff
@@ -293,13 +297,22 @@ func loadArtifact(artifactBkt *bolt.Bucket, digest string) (*ArtifactEntry, erro
 	if err != nil {
 		return nil, err
 	}
+	createdAt := time.Now()
```

Comment:
Do we want to default to `time.Now()`? That would mean indices created before this change would always show up as the newest ones, right?

Reply:
Yes, I'm not sure what a good default is in this case? Could use `*time.Time` instead. Honestly, I don't think maintaining backwards compatibility here is really important.

Reply:
Fair point about backwards compatibility. I think `time.Time{}` might be more defensive. If we do have some sort of regression, 0001-01-01 is pretty easy to spot and I can imagine some subtly frustrating bugs if your index crated time keeps changing to be this very instant.

Reply:
Done.